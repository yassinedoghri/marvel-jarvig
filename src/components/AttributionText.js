import Paragraph from "components/Paragraph";

const AttributionText = Paragraph.extend`
  color: ${props => props.theme.colors.contrastLight.disabled};
  font-weight: bold;
  width: 100%;
  text-align: center;
  padding: ${props => props.theme.spaces.md}em 0;
`;

export default AttributionText.withComponent("footer");
