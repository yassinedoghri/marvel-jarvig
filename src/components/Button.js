import { transitions } from "polished";
import styled from "styled-components";
import media from "utils/media";
import { getColor, setFontSize } from "utils/style";

const Button = styled.button`
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: ${props => (props.iconright ? "row-reverse" : "row")};
  color: ${props => getColor(props.theme, props.color, props.tint)};
  background-color: ${props => getColor(props.theme, props.backgroundcolor)};
  border: none;
  cursor: pointer;
  text-align: center;
  border-radius: 0.5em;
  padding: ${props => props.theme.spaces.sm}em
    ${props => props.theme.spaces.md}em;
  box-shadow: 0 2px 0
      ${props => getColor(props.theme, props.backgroundcolor, "dark")},
    inset 0 1px 0 rgba(255, 255, 255, 0.3),
    inset 0 0 3px rgba(255, 255, 255, 0.5), inset 0em 0em 2em rgba(0, 0, 0, 0.3);
  ${props => setFontSize(props.theme)};
  font-family: "Bangers", sans-serif;
  letter-spacing: 0.1em;
  text-transform: uppercase;
  ${transitions("all 300ms ease-in-out")};

  &:hover {
    box-shadow: 0 2px 0
        ${props => getColor(props.theme, props.backgroundcolor, "dark")},
      inset 0 1px 0 rgba(255, 255, 255, 0.3),
      inset 0 0 3px rgba(255, 255, 255, 0.5),
      inset 0em 0em 1em rgba(0, 0, 0, 0.3);
  }

  &:active {
    box-shadow: 0 2px 0
        ${props => getColor(props.theme, props.backgroundcolor, "dark")},
      inset 0 1px 0 rgba(255, 255, 255, 0.3),
      inset 0 0 3px rgba(255, 255, 255, 0.5),
      inset 0em 0em 3em rgba(0, 0, 0, 0.3);
  }

  &:disabled,
  &[disabled] {
    background: ${props => props.theme.colors.default.base};
    color: ${props => props.theme.colors.default.dark};
    box-shadow: none;
    cursor: not-allowed;
  }

  ${media.desktop`
    padding: ${props => props.theme.spaces.md}em
    ${props => props.theme.spaces.lg}em;
  `};

  & span {
    font-family: "Bangers", sans-serif;
  }

  & svg {
    margin: 0 ${props => (props.iconright ? 0 : `${props.theme.spaces.sm}em`)} 0
      ${props => (props.iconright ? `${props.theme.spaces.sm}em` : 0)};
  }
`;

export default Button;
