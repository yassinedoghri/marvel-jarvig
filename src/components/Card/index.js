import ActionBtn from "components/Card/ActionBtn";
import ActionContainer from "components/Card/ActionContainer";
import ActionLink from "components/Card/ActionLink";
import BodyContainer from "components/Card/BodyContainer";
import BodyParagraph from "components/Card/BodyParagraph";
import BodyResult from "components/Card/BodyResult";
import Hint from "components/Card/Hint";
import HintMessage from "components/Card/HintMessage";
import Ribbon from "components/Card/Ribbon";

import Title from "components/Card/Title";
import { bounceIn, shake, tada } from "react-animations";
import styled, { keyframes } from "styled-components";
import { media } from "utils";

const tadaAnimation = keyframes`${tada}`;
const bounceInAnimation = keyframes`${bounceIn}`;
const shakeAnimation = keyframes`${shake}`;

const Card = styled.div`
  position: relative;
  max-width: 30em;
  width: 100%;
  background-color: ${props => props.theme.colors.background.base};
  border: solid ${props => props.theme.colors.foreground.base};
  border-width: 3px 4px 3px 5px;
  border-radius: 95% 4% 92% 5%/4% 95% 6% 95%;
  box-shadow: 0 3px 15px rgba(0, 0, 0, 0.2);
  padding: ${props => props.theme.spaces.md}em;
  margin: 0 auto ${props => props.theme.spaces.lg}em;
  animation: 1s
    ${props => {
      if (props.animation === "correct") return tadaAnimation;
      if (props.animation === "incorrect") return shakeAnimation;
      return bounceInAnimation;
    }};

  ${media.desktop`
      width: auto;
      margin-bottom: 0;
  `};
`;

Card.Title = Title;
Card.Ribbon = Ribbon;
Card.BodyParagraph = BodyParagraph;
Card.BodyResult = BodyResult;
Card.BodyContainer = BodyContainer;
Card.ActionBtn = ActionBtn;
Card.ActionLink = ActionLink;
Card.ActionContainer = ActionContainer;
Card.Hint = Hint;
Card.HintMessage = HintMessage;

export default Card;
