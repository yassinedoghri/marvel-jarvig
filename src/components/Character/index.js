import Caption from "components/Character/Caption";
import CheckBadge from "components/Character/CheckBadge";

import Figure from "components/Character/Figure";
import Image from "components/Character/Image";
import Label from "components/Character/Label";
import RadioBtn from "components/Character/RadioBtn";
import styled from "styled-components";
import { getResultColor } from "utils/style";

const Character = styled.li.attrs({
  // or we can define dynamic ones
  resultcolor: props => getResultColor(props.theme, props.result)
})`
  margin: 0 auto;
  flex: 0 0 auto;

  ${Image} {
    border-color: ${props => props.resultcolor};
    border-style: solid;
  }

  ${CheckBadge} {
    background: ${props => props.resultcolor};
    border-color: ${props => props.resultcolor};
    border-style: solid;
  }

  ${Caption} {
    border-color: ${props => props.resultcolor};
    background: ${props => props.resultcolor};
    color: ${props => props.theme.colors.contrastDark.base};
  }

  ${RadioBtn} {
    &:disabled + label > figure > img,
    &:disabled + label > figure > figcaption {
      border-color: ${props => props.resultcolor};
    }

    &:disabled + label > figure > figcaption {
      background: ${props => props.resultcolor};
    }
  }
`;

Character.Figure = Figure;
Character.Image = Image;
Character.Caption = Caption;
Character.CheckBadge = CheckBadge;
Character.RadioBtn = RadioBtn;
Character.Label = Label;

export default Character;
