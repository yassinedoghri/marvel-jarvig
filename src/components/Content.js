import styled from "styled-components";

export const Content = styled.section`
    background-color: ${props => props.theme.colors.primary.base};
    color: ${props => props.theme.colors};;
    display: flex;
    flex-direction: column;
    flex-grow: 2;
    font-size: large;
    padding: 4em 2em; 
`;

export default Content;
