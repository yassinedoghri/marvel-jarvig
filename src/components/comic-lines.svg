<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" viewBox="0 0 1620 1080">
	<style>
		path{fill:#980c11}
	</style>
	<path d="M1620 419.2v18.6c-183 18.7-367.3 36.7-367.7 33-.4-3.6 184.5-28.2 367.7-51.6z"/>
	<path d="M1620 439.3V494c-197.1 20.3-414.5 41.8-414.9 37.9-.4-3.3 151.1-23.7 315.6-45-129.6 12.8-234 22.3-234.3 19.6-.4-3.3 148.8-23.4 311.8-44.5-198.5 20.5-419.9 42.4-420.3 38.5-.4-4.2 238.5-35.4 442.1-61.2z"/>
	<path d="M1620 324v37.9C1472.5 394 1344.4 421 1343.7 418c-.4-1.8 47.6-14.9 118.8-33.1-66.5 13.9-110.9 22.5-111.3 20.7-.1-.5 3.4-1.8 10-3.9-107.8 23.1-189.3 39.8-189.8 37.4-1.1-4.1 245.9-65.5 448.6-115.1z"/>
	<path d="M1305 346.2c.7 1.9 54.4-15.3 131.6-41.1-109.5 41.3-193.6 73.8-192.8 76.2.9 2.6 100.6-30.3 224-72.3-76.6 29.2-129.4 50.1-128.7 52 1.1 3 132.7-41 280.9-91.6v-45.2c-162.1 60.4-316.1 118.8-315 122z"/>
	<path d="M1620 110.1v25c-195.6 93.2-487.3 231.2-489.4 226.9-2.2-4.3 295.9-154.8 489.4-251.9z"/>
	<path d="M1620 135.4v24.4c-194.5 92.6-462.4 219.1-464.4 215-2.1-4.3 271.3-142.5 464.4-239.4z"/>
	<path d="M1620 1.5V43c-177.4 110.5-514.4 319.4-517.3 314.9-1.3-2 64-46.1 152.2-104.4-62.5 37.8-105.1 62.8-106.1 61.2-2.7-4.3 290.3-195.7 471.2-313.2z"/>
	<path d="M1519 0c-168.6 132.3-394.6 308.3-397.4 304.8-1.8-2.2 86.9-76.8 194.3-165.7-109.3 84.9-200.3 154.5-202.1 152.3-1.6-2 69.9-62.5 162.5-139.3-43.3 32.9-71.4 53.5-72.4 52.3-2.1-2.6 114.1-99.5 241.3-204.4h73.8z"/>
	<path d="M1334.2 0c-131.8 128.5-265.3 257.4-267.9 254.8-2.5-2.5 117.9-127.4 242.1-254.8h25.8z"/>
	<path d="M1360.5 0c-79 76.2-139.1 133.1-140.8 131.4-1.7-1.6 50.7-57.1 122.3-131.4h18.5z"/>
	<path d="M1219.1 0c-54.4 65.1-93 110.1-94.6 108.9-1.5-1.2 31.5-45.4 80.2-108.9h14.4z"/>
	<path d="M1242 0c-62.4 74.9-108.4 128.8-110.1 127.4-1.7-1.4 38.1-54.2 94.5-127.4h15.6z"/>
	<path d="M1152.5 0C1065.4 132.9 986.7 251.2 984 249.5c-1.6-1 23-43 60.4-104.8-35.7 53.4-59.8 88.2-61.3 87.3-2.5-1.6 61.7-107.7 138.1-232h31.3z"/>
	<path d="M1051.5 0c-56.8 111.4-102.4 199.1-104.8 198-2.3-1.1 37.3-87.7 89-198h15.8z"/>
	<path d="M1073.9 0c-30.6 59-51.4 97.7-53 97-1.6-.8 15.3-39.1 42-97h11z"/>
	<path d="M996.9 0c-39.9 105.8-71.1 186.1-73.4 185.3-2.3-.8 23.7-80.4 59.2-185.3h14.2z"/>
	<path d="M928.7 0c-29 115.6-52.3 205-54.8 204.4-2.5-.6 15.8-89.5 40.5-204.4h14.3z"/>
	<path d="M949.4 0c-27.6 109.8-49.4 193.1-51.8 192.6-2.4-.6 14.6-83.4 38-192.6h13.8z"/>
	<path d="M851.7 0h9.5c-8.4 59.8-14.4 98.6-16.1 98.4-1.6-.2 1.2-38.9 6.6-98.4z"/>
	<path d="M877.5 0c-7.8 55.6-13.5 91.2-15.1 91-1.6-.2 1-35.7 5.9-91h9.2z"/>
	<path d="M789.8 0h8.5c-1.3 49-2.7 79.8-4.2 79.8S791.2 49 789.8 0z"/>
	<path d="M814.2 0c-1.6 71.1-3.3 118.8-5.2 118.8-1.8 0-3.6-47.7-5.2-118.8h10.4z"/>
	<path d="M728.5 0c7.6 80.4 12.1 135.9 10.1 136.2-2 .2-10.3-55.5-21.3-136.2h11.2z"/>
	<path d="M770.7 220.9c-2.6.3-16.3-97.3-32.6-220.9h14.4c12.1 123.3 20.8 220.6 18.2 220.9z"/>
	<path d="M672.8 0h6.1c4.7 24.6 7.1 39 6.1 39.2-1 .2-5.4-14.2-12.2-39.2z"/>
	<path d="M703.7 171c-2.3.5-21.4-72.1-46.4-171h13c21 98.3 35.6 170.5 33.4 171z"/>
	<path d="M585.3 0c31.3 92.9 53.3 160.7 51.1 161.5-2.1.8-28.9-67.7-64.4-161.5h13.3z"/>
	<path d="M597.8 0c26.1 78 43.5 132.3 41.6 133-1.9.7-23.6-54.1-53.6-133h12z"/>
	<path d="M679.7 187.8c-2.4.8-34-80.7-74.4-187.8h14.3c35.9 106.2 62.4 187 60.1 187.8z"/>
	<path d="M524.7 0h14.8c46.3 99 80.3 173.8 78.2 174.9-2.2 1.1-41.8-74.7-93-174.9z"/>
	<path d="M643.2 271c-2.9 1.4-69.2-127.5-141.9-271h18.6c67.2 142.5 126.2 269.6 123.3 271z"/>
	<path d="M448.8 0c89 144.2 171.4 279.6 168.6 281.4C614.5 283.2 523 145.2 427.9 0h20.9z"/>
	<path d="M640.4 283.7C637.5 285.5 545.1 146 449.5 0h20.9c89.5 145 172.8 281.9 170 283.7z"/>
	<path d="M332.7 0c71 91.8 125 162.8 123 164.3-2 1.6-63.3-70.9-140.8-164.3h17.8z"/>
	<path d="M500.7 186.6c-2.2 1.8-73.2-82.7-159.2-186.6h32.6c55.9 72.6 95.3 124.9 93.6 126.3-1.5 1.2-34.6-37.3-82.6-94.5 67.3 87 117.5 153.3 115.6 154.8z"/>
	<path d="M545.1 307.4c-1.5 1.5-49.4-43.5-115.7-107.2 39.6 41.7 64.8 69 63.7 70.2-2.7 2.6-147-136.9-283.8-270.4h42.5c139.9 143.2 296.1 304.5 293.3 307.4z"/>
	<path d="M269.8 153.1c122.4 101 231.2 192 229.2 194.5C495.9 351.5 216.5 133 47.2 0h89.9c148.8 122.3 306 252.8 303.7 255.8-1.8 2.3-98.4-71.6-212.1-160C383 222.5 554.9 364.9 552.4 368c-2.2 2.8-141.4-104.7-282.6-214.9z"/>
	<path d="M407.4 316.6C405 320.3 179.3 181.3 0 70.1V45c178.5 116.2 409.7 267.8 407.4 271.6z"/>
	<path d="M0 170v-19.5c153.6 77.7 298.8 152.1 297.4 155.2C295.9 308.8 152.8 242.2 0 170z"/>
	<path d="M459.4 407C457.4 411.1 194 286.7 0 194.4v-24.3C192.8 266.9 461.5 402.8 459.4 407z"/>
	<path d="M119.1 322.5c117.5 44.2 210.3 80 209.4 82.5-1.2 3.3-162.1-50.9-328.5-107.9v-38c199.1 73.6 454 168.9 452.5 173-1.2 3.3-165.2-52-333.4-109.6z"/>
	<path d="M267.2 462.9c-.7 2.9-123.5-22.9-267.2-54.2v-16.3c144.2 35.9 267.9 67.5 267.2 70.5z"/>
	<path d="M475.1 483.4c-.6 2.4-87.5-15.4-200.6-39.7 27.1 7.3 43 12.1 42.7 13.2-.8 3.3-153.2-29.2-317.2-65v-38c171.2 42.3 336.6 84.1 335.8 87.5-.2 1-14.5-1.3-38.8-6 102.6 25.7 178.6 45.7 178.1 48z"/>
	<path d="M422.4 528.5c-.3 2.5-89.8-5.4-206-16.8 86.6 11.7 147.6 20.7 147.4 22.8-.4 3.6-182.1-14.1-363.8-32.7v-32c199 25.4 422.8 54.8 422.4 58.7z"/>
	<path d="M284.4 565.8c68.1 1.6 113.4 3.3 113.4 5.1 0 2.4-83.2 4.7-193.5 6.7 24.3.9 38.4 1.9 38.4 2.9 0 2.8-108.9 5.4-242.7 7.5v-39.2c211 2.3 476.5 6.2 476.5 10.5 0 2.3-82.5 4.6-192.1 6.5z"/>
	<path d="M0 670.9v-31.7c210.2-21.9 476.2-48.6 476.7-44.3.4 3.2-141.4 22.4-299.6 42.9 58.2-5.3 96-8 96.2-6.4.4 2.9-126 20.3-273.3 39.5z"/>
	<path d="M497.6 626.4c.5 4.5-284.8 41.4-497.6 68.2v-21.7c213-22.3 497.1-51 497.6-46.5z"/>
	<path d="M0 766.7v-18.4c171.9-37.7 337.9-73.1 338.7-69.7.8 3.3-166.5 45.6-338.7 88.1z"/>
	<path d="M424.6 678.6c.9 4-226.9 60.8-424.6 109.3v-20.7c197.6-43.6 423.7-92.6 424.6-88.6z"/>
	<path d="M57 841.9c118.7-40.3 212.9-71.3 213.9-68.8C272 776 144.9 824.5 0 878.6V844c167.1-57.3 329.3-111.9 330.5-108.6 1 2.9-127.6 51.9-273.5 106.5z"/>
	<path d="M542.9 718.7c2.3 4.8-357.2 185.7-542.9 278.7v-56.9c193.4-92 451.8-214 453.8-209.9.8 1.6-36.4 21.6-93.4 51.1 23.8-10.6 37.9-16.3 38.4-15.4 1.6 3.4-173 92.5-341.8 177.7 195.5-93.2 483.8-229.6 485.9-225.3z"/>
	<path d="M308.2 908c38.9-23.2 63.4-37.2 64.1-36 2 3.1-154.8 106.6-309.8 208H0v-12.4c184-114.3 456.1-282.4 458.7-278.4 1 1.6-38.1 28.5-97 67.8 92.2-56.3 161.8-97.9 163.1-95.9 1.6 2.4-97.7 68.7-216.6 146.9z"/>
	<path d="M186.2 1080h-30.5c153.5-120.1 319.4-248.7 321.9-245.6 2.3 2.8-146.3 126.3-291.4 245.6z"/>
	<path d="M219.6 1080h-29c144.7-113 290.6-225.9 292.9-223 2.1 2.7-128.2 111.2-263.9 223z"/>
	<path d="M494.9 913.6c1.9 1.9-67.9 75.2-156.2 166.4h-43.8C436.7 941.5 594 789.3 596.8 792.1c2.2 2.2-94.4 102.9-203.9 215.6 59-56.8 100.5-95.6 102-94.1z"/>
	<path d="M552.7 938c1.8 1.4-43.4 61.3-105.7 142h-57.5C486.9 962.1 572.6 859.8 575 861.7c1.9 1.5-47.4 66.6-113.9 152.6 36.6-43.4 60.7-70.9 61.9-69.9.8.7-8.4 13.8-24.7 35.7 60.3-72.3 104.3-123.9 106-122.5 1.7 1.4-39.6 56.3-97.8 131.8 27.6-32.5 45.1-52.3 46.2-51.4z"/>
	<path d="M535.8 1055.6c.7.4-3.9 9.2-12.6 24.4h-6c11.2-15.9 17.9-24.8 18.6-24.4z"/>
	<path d="M501.2 1080h-14.1c51.3-77.4 88.7-132.4 90.6-131.2 1.8 1.1-30.3 55.2-76.5 131.2z"/>
	<path d="M587.5 1080h-16.2c58.8-115.5 106.8-208 109.3-206.8 2.4 1.2-39.4 92.4-93.1 206.8z"/>
	<path d="M662.9 1045.9c.9.3-2.6 12.8-9.2 34.1h-6c8.8-21.9 14.3-34.4 15.2-34.1z"/>
	<path d="M638.3 1080h-9.8c21.2-54.9 35.5-90.3 37-89.7 1.5.5-9.4 35.6-27.2 89.7z"/>
	<path d="M699 1080h-11.1c19.2-75 33-126.1 34.9-125.7 1.8.5-8.2 51.3-23.8 125.7z"/>
	<path d="M712 1080h-11c18.8-73.2 32.2-122.9 34-122.5 1.9.5-7.8 49.9-23 122.5z"/>
	<path d="M767.9 1077.9c.2 0 .4.7.4 2.1H767c.3-1.4.6-2.1.9-2.1z"/>
	<path d="M788.2 1080h-12.5c13-97.1 23.3-167.6 25.5-167.3 2.3.3-3.7 70.5-13 167.3z"/>
	<path d="M856.8 1080h-12.4c1.8-98.3 4-169.9 6.2-169.9s4.4 71.6 6.2 169.9z"/>
	<path d="M896.6 1080H878c-7.3-78.1-11.6-131.8-9.7-132 1.9-.2 9.9 52.9 20.5 130.5-3.3-38.7-4.7-62.4-3.4-62.5 1.3-.2 5.4 24.2 11.2 64z"/>
	<path d="M906.4 1080h-5.8c-1.8-23.4-2.4-37-1.4-37.1 1-.1 3.6 13.5 7.2 37.1z"/>
	<path d="M944.7 1080h-14.1c-24.3-113-42.1-199.8-39.7-200.4 2.5-.6 25.3 86.8 53.8 200.4z"/>
	<path d="M970 1080h-9.9c-12.7-61.1-20.4-101.3-18.7-101.7 1.6-.4 12.7 40 28.6 101.7z"/>
	<path d="M1015.2 1080h-4c-2.9-10-4.2-15.5-3.6-15.7.5-.3 3.2 5.3 7.6 15.7z"/>
	<path d="M1036.8 1080H1021c-42.4-124.8-75.8-226-73.3-226.9 2.7-1 41.9 101.2 89.1 226.9z"/>
	<path d="M1088 1080h-12.1c-32.1-69.5-53.4-117.2-51.7-118 1.7-.9 27.4 47.4 63.8 118z"/>
	<path d="M1120.9 1080h-28c-56.8-121-102.3-219.8-99.8-221 1.6-.8 23.3 39.6 54.9 100.6-27.7-60.1-45.5-100.3-43.9-101 2.6-1.3 54.6 99.1 116.8 221.4z"/>
	<path d="M1178.4 1080h-24c-47.8-78.5-81.2-134.9-79.4-136.1 1.8-1.2 40.1 55.1 92.4 134-22.5-37.7-35.9-61.4-34.8-62.1 1.3-.8 18.7 24.1 45.8 64.2z"/>
	<path d="M1193.8 1080h-9.7c-23.1-38.7-37-63.1-35.8-63.9 1.2-.8 18.6 24 45.5 63.9z"/>
	<path d="M1288.6 1080h-36c-54.4-70.7-92.3-121.2-90.7-122.5 1.5-1.2 33.5 36 80.2 91.7-93.7-120.4-174.7-226-172.3-227.9 2.6-2.2 108.5 124.8 218.8 258.7z"/>
	<path d="M1375.8 1080h-23.4c-106.9-110-199.7-206.9-197.6-209.1 2.3-2.2 106.2 97.5 221 209.1z"/>
	<path d="M1487.3 1080h-30.8c-146.4-120.4-298-246.3-295.7-249.2 2.5-3.1 171.7 128.1 326.5 249.2z"/>
	<path d="M1521.1 1080h-32.7c-157-128.9-337.1-278-334.5-281.2 2.6-3.3 203 152.6 367.2 281.2z"/>
	<path d="M1620 1020.4v48.6c-176.8-115.2-398.6-260.7-396.3-264.4.7-1.2 24.3 12.2 61.9 34.6-107-70.5-192.1-127.6-190.7-129.8.9-1.3 31.8 16.5 79.4 45.1-24.8-17-39.2-27.4-38.6-28.4 2.8-4.2 301.2 180.4 484.3 294.3z"/>
	<path d="M1620 894.3v55.6c-190.1-95.6-436-220.1-434-224.1 1.6-3.3 178 79.1 348.3 159.8-128.6-65.3-237-121.3-235.7-123.9.7-1.5 36.4 14.2 90.9 39.2-106.7-54.4-189.4-97.5-188.2-99.8 1.8-3.9 230.5 103.8 418.7 193.2z"/>
	<path d="M1620 805.3v21.8c-194.8-72.1-429.5-160-428.1-164 1.4-3.8 233.1 75 428.1 142.2z"/>
	<path d="M1620 699.9v46.5c-199.9-49-435.8-107.8-434.9-111.8.6-2.8 115.5 21.2 252.9 51-106.4-26.8-186.4-47.8-185.9-50.1.3-1.3 24 3 62.5 10.7-99.6-25.1-172.9-44.5-172.3-46.7 1-4.3 270 54.4 477.7 100.4z"/>
	<path d="M1620 598.3v20c-199-25.3-422.6-54.7-422.2-58.6.5-4 223.3 18.1 422.2 38.6z"/>
	<path d="M1620 619.9v27c-206.8-26.2-458-58.9-457.5-63.1.3-2.7 104.1 6.7 233.1 19.5-8.8-1.6-13.5-2.7-13.5-3.3.4-2.8 106.7 6.9 237.9 19.9z"/>
</svg>