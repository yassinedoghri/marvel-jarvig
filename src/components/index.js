export { default as Footer } from "components/Footer";
export { default as Header } from "components/Header";
export { default as Main } from "components/Main";
export { default as Button } from "components/Button";
export { default as Paragraph } from "components/Paragraph";
export { default as HR } from "components/HR";
export { default as Logo } from "components/Logo";
export { default as Toolbar } from "components/Toolbar";
export { default as FlexWrapper } from "components/FlexWrapper";
export { default as Sidebar } from "components/Sidebar";
export { default as Settings } from "components/Settings";
export { default as FlexGrid } from "components/FlexGrid";
export { default as PlayButton } from "components/PlayButton";
export { default as FlexSection } from "components/FlexSection";
export { default as Character } from "components/Character";
export { default as Card } from "components/Card";
export { default as PlaySection } from "components/PlaySection";
export { default as GameUI } from "components/GameUI";
export { default as Emphasis } from "components/Emphasis";
export { default as PlayMode } from "components/PlayMode";
export { default as AttributionText } from "components/AttributionText";
export { default as Loader } from "components/Loader";
export { default as ChoicesSection } from "components/ChoicesSection";
