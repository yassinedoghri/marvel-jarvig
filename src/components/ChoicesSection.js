import { FlexSection } from "components";
import media from "utils/media";

const ChoicesSection = FlexSection.extend`
  align-items: center;
  justify-content: center;
  max-width: 70em;
  padding: 0;

  ${media.desktop`
      padding: initial;
  `};
`;

export default ChoicesSection;
