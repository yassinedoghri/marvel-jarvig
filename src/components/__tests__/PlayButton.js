import { PlayButton } from "components";
import React from "react";
import { MemoryRouter } from "react-router-dom";
import { renderWithTheme } from "utils/testHelpers";

describe("PlayButton", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      <MemoryRouter>
        <PlayButton to="/" />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
