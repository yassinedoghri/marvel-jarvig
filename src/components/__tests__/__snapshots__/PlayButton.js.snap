// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayButton renders correctly 1`] = `
<a
  className="sc-ckVGcZ eZeqXi"
  href="/"
  onClick={[Function]}
/>
`;
